feature-libs/cart/base/components/added-to-cart-dialog/added-to-cart-dialog.component.html

```diff
@@ -1,4 +1,4 @@
-<div #dialog>
+<div #dialog class="AddedToCartDialogComponent">
```

Comment:
Why do you add this class? I don't see usage. 
Additionally, it doesn't comply with our convention (`cx-added-to-cart-dialog`)

Reply:
That's a mistake.  Not sure wht its there & is removed.

Reply:
Yeah, I also don't know why it is still here 😅